File: app/src/main/java/com/arnaudpiroelle/manga/provider/mangapanda/api/MangaPandaApiService.java
package com.arnaudpiroelle.manga.provider.mangapanda.api;

import retrofit.client.Response;
import retrofit.http.GET;
import retrofit.http.Path;

public interface MangaPandaApiService {
    @GET("/alphabetical")
    Response getMangaList();

    @GET("/{mangaAlias}")
    Response getManga(@Path("mangaAlias") String mangaAlias);

    @GET("/{mangaAlias}/{chapterNumber}")
    Response getReader(@Path("mangaAlias") String mangaAlias, @Path("chapterNumber") String chapterNumber);

    @GET("/{mangaAlias}/{chapterNumber}/{pageNumber}")
    Response getPage(@Path("mangaAlias") String mangaAlias, @Path("chapterNumber") String chapterNumber, @Path("pageNumber") String pageNumber);
}

File: temp/mangapanda/api/MangaPandaApiService.kt
package com.arnaudpiroelle.manga.provider.mangapanda.api

import retrofit.client.Response
import retrofit.http.GET
import retrofit.http.Path

interface MangaPandaApiService {
    @GET("/alphabetical")
    fun getMangaList(): Response

    @GET("/{mangaAlias}")
    fun getManga(@Path("mangaAlias") mangaAlias: String): Response

    @GET("/{mangaAlias}/{chapterNumber}")
    fun getReader(@Path("mangaAlias") mangaAlias: String, @Path("chapterNumber") chapterNumber: String): Response

    @GET("/{mangaAlias}/{chapterNumber}/{pageNumber}")
    fun getPage(@Path("mangaAlias") mangaAlias: String, @Path("chapterNumber") chapterNumber: String, @Path("pageNumber") pageNumber: String): Response
}
